### DISCUSSION TITLE
Export data to CSV

### DISCUSSION THREAD

**Discussion by mydigitalbubble** [tnmendes/watt#78] [Q&A] [answered]
I see there is reference to exporting the consumption history to CSV or Excel. Can someone guide me on how this is done? I cant see the option in the IOS app - or is it still a pending feature enhancement?

thanks

**Comment by tnmendes** [comment]
Hey mydigitalbubble,

At the moment it is not yet possible to export, it is in the Roadmap (planned to develop in the future)

**Comment by tnmendes** [comment, marked as answer]
You can now export CSV with the new version Watt 5.0
